charts/posthog/templates/clickhouse_instance.yaml

```diff
@@ -13,6 +13,9 @@ spec:
 
     profiles:
       default/allow_experimental_window_functions: "1"
```

Comment:
As discussed on Slack, let's put this as a common default in `values.yaml` and then merge that map/dict with `.Values.clickhouse.profiles`